primitives/ethereum-primitives/src/header.rs

```diff
@@ -174,7 +174,10 @@ impl EthereumHeader {
 					array_bytes::hex2array_unchecked!(parse_value_unchecked(s), 32).into();
 			}
 		}
-		eth_header.seal = vec![rlp::encode(&mix_hash), rlp::encode(&nonce)];
+		eth_header.seal = vec![
+			rlp::encode(&mix_hash).to_vec(),
```

Comment:
Is there difference between `rlp::encode(&mix_hash)` and `rlp::encode(&mix_hash).to_vec()` ?

Reply:
`rlp::encode` now returns a bytes thing, we've to convert it to `Vec` manually.